app/policies/synapse_policy.rb

```diff
@@ -16,16 +16,11 @@ def index?
   end
 
   def create?
```

Comment:
There is always there fact that the permission of the topics which a synapse connects were accessible at the time it was created and then get changed, so we still have to bear those scenarios